core/src/main.rs

```diff
@@ -104,29 +103,20 @@ fn main() {
     logging::setup_log();
 
     let args = Args::parse();
-    let (_, repost_receiver) = bounded(1);
+    //let (_, repost_receiver) = bounded(1);
 
     // Start Oracle Core GET API Server
-    thread::Builder::new()
-        .name("Oracle Core GET API Thread".to_string())
-        .spawn(|| {
-            api::start_get_api(repost_receiver);
-        })
-        .ok();
+    //thread::Builder::new()
+    //    .name("Oracle Core GET API Thread".to_string())
```

Comment:
I think we can do away with the `api` module directly. What do you think?

Reply:
Sure. The API instantiates instances of `OraclePool` which means it needs an existing oracle pool YAML config file. Therefore it makes sense to move it under the CLI `Command::Run` as I've done with the latest commit.